python/pyspark/sql/functions.py

```diff
@@ -91,13 +92,48 @@ def lit(col):
     Creates a :class:`Column` of literal value.
```

Comment:
Worth mentioning the supported types in the docs

Reply:
Added a short description of the input parameters @LielBach.